### ISSUE TITLE
adcprep issue with global mesh and NOAA tagged of ADCIRC

### ISSUE THREAD

**Issue by GStefK** [StormSurgeLive/asgs#1044]
I got this error by running adcprep and prepall

```
Using default filenames.

 calling: prepinput
 use_default =  T
 partition =  F
 prep_all  =  T
 prep_15   =  F
 prep_13   =  F
 hot_local  =  F
 hot_global  =  F
 File                                                      fort.14
 WAS FOUND!  Opening & Processing file

 File fort.15
 WAS FOUND!  Opening & Processing file

 INFO: The SWANOutputControl namelist was not found.
 Elevation Station Locations contained in fort.15
forrtl: severe (59): list-directed I/O syntax error, unit 15, file /ddnA/work/sgiaremi/asgs45406/initialize/hindcast/fort.15
Image              PC                Routine            Line        Source
adcprep            00000000006CAA7B  for__io_return        Unknown  Unknown
adcprep            0000000000701AAB  for_read_seq_lis_     Unknown  Unknown
adcprep            0000000000700315  for_read_seq_lis      Unknown  Unknown
adcprep            0000000000411989  Unknown               Unknown  Unknown
adcprep            000000000048354A  Unknown               Unknown  Unknown
adcprep            0000000000482FDA  Unknown               Unknown  Unknown
adcprep            000000000040DD62  Unknown               Unknown  Unknown
libc-2.17.so       00002BA16A339555  __libc_start_main     Unknown  Unknown
adcprep            000000000040DC69  Unknown               Unknown  Unknown
```

```json
  {
    "adcirc.build.swan.macros-inc"             : "/ddnA/work/$USER/asgs/opt/models/adcircs/adcirc-cg-noaa.stofs.2d.glo.v1.1.0r2-intel-supermic/thirdparty/swan/macros.inc",
    "adcirc.build.swan.macros-inc.F90_SER"     : "ifort",
    "adcirc.build.swan.macros-inc.F90_MPI"     : "mpif90",
    "adcirc.build.swan.macros-inc.F90_OMP"     : "ifort",
    "adcirc.build.swan.macros-inc.FLAGS_OPT"   : "-O2",
    "adcirc.build.swan.macros-inc.FLAGS_MSC"   : "-W0 -assume byterecl -traceback -diag-disable 8290 -diag-disable 8291 -diag-disable 8293",
    "adcirc.build.swan.macros-inc.FLAGS90_MSC" : "$(FLAGS_MSC)",
    "adcirc.build.swan.macros-inc.FLAGS_SER"   : "",
    "adcirc.build.swan.macros-inc.FLAGS_MPI"   : "",
    "adcirc.build.swan.macros-inc.FLAGS_OMP"   : "-qopenmp",
    "time.adcirc.executables.built"      : "2022-Nov-18-T18:21:23-0600",
    "adcirc.source.commit"               : "6bf1d614a90e545bd4af0d4f57272fec34e63415",
    "adcirc.source.asgs.patches.set"     : "noaa.stofs.2d.glo.v1.1.0r2",
    "adcirc.source.asgs.patches.applied" : [
      "/ddnA/work/$USER/asgs/patches/ADCIRC/noaa.stofs.2d.glo.v1.1.0r2/00-cmplrflags-mk.patch"
    ],
    "adcirc.source.branch-base"           : "noaa.stofs.2d.glo.v1.1.0r2",
    "env.adcirc.build.ASGS_HOME"          : "/ddnA/work/$USER/asgs",
    "env.adcirc.build.ASGS_MACHINE_NAME"  : "supermic",
    "env.adcirc.build.NETCDFHOME"         : "/ddnA/work/$USER/asgs/opt",
    "env.adcirc.build.ADCIRCBASE"         : "/ddnA/work/$USER/asgs/opt/models/adcircs/adcirc-cg-noaa.stofs.2d.glo.v1.1.0r2-intel-supermic",
    "env.adcirc.build.ADCIRCDIR"    : "/ddnA/work/$USER/asgs/opt/models/adcircs/adcirc-cg-noaa.stofs.2d.glo.v1.1.0r2-intel-supermic/work",
    "env.adcirc.build.SWANDIR"            : "/ddnA/work/$USER/asgs/opt/models/adcircs/adcirc-cg-noaa.stofs.2d.glo.v1.1.0r2-intel-supermic/thirdparty/swan",
    "env.adcirc.build.CUSTOM_SRC"         : "0", 
    "env.adcirc.build.ADCIRC_COMPILER"    : "intel",
    "env.adcirc.build.ADCIRC_GIT_BRANCH"  : "noaa.stofs.2d.glo.v1.1.0r2",
    "env.adcirc.build.ADCIRC_GIT_URL"     : "git@github.com:adcirc",
    "env.adcirc.build.ADCIRC_GIT_REPO"    : "adcirc-cg",
    "env.adcirc.build.ADCIRC_SRC_TYPE"    : "git",
    "env.adcirc.build.ADCIRC_BASE_URL"    : "0",
    "env.adcirc.build.ADCIRC_SRC_FILE"    : "0",
    "env.adcirc.build.ADCIRC_EXTRACT_DIR" : "0",
    "env.adcirc.build.ADCIRC_ARCHIVE_URL" : "0",
    "env.adcirc.build.ASGS_MAKEJOBS"      : "1",
    "env.adcirc.build.ADCIRC_MAKE_CMD"    : "make padcirc adcirc adcprep hstime aswip SWAN=enable compiler=intel NETCDF=enable NETCDF4=enable NETCDF4_COMPRESSION=enable NETCDFHOME=/ddnA/work/$USER/asgs/opt MACHINENAME=supermic",
    "env.adcirc.build.SWAN_UTIL_BINS_MAKE_CMD" : "make unhcat compiler=intel MACHINENAME=supermic",
    "env.adcirc.build.ADCSWAN_MAKE_CMD"        : "make adcswan padcswan compiler=intel MACHINENAME=supermic NETCDF=enable NETCDF4=enable NETCDF4_COMPRESSION=enable NETCDFHOME=/ddnA/work/$USER/asgs/opt",
    "env.adcirc.build.ADCIRC_PROFILE_NAME"     : "noaa.stofs.2d.glo.v1.1.0r2-intel-supermic",
    "env.adcirc.build.ADCIRC_BINS"        : "padcirc adcirc adcprep hstime aswip",
    "env.adcirc.build.ADCSWAN_BINS"       : "adcswan padcswan",
    "env.adcirc.build.SWAN_UTIL_BINS"     : "unhcat.exe",
    "env.adcirc.build.PATH"               : "/ddnA/work/$USER/asgs/opt/perl5/bin:/ddnA/work/$USER/asgs/opt/perl5/perls/perl-5.34.0/bin:/ddnA/work/$USER/asgs/opt/bin:/ddnA/work/$USER/asgs/cloud/general:/ddnA/work/$USER/asgs/bin:/ddnA/work/$USER/asgs/my-bin:/ddnA/work/$USER/asgs/cloud/general:/ddnA/work/$USER/asgs/config:/ddnA/work/$USER/asgs/config/tests:/ddnA/work/$USER/asgs/input:/ddnA/work/$USER/asgs/input/data_assimilation:/ddnA/work/$USER/asgs/monitoring:/ddnA/work/$USER/asgs/output:/ddnA/work/$USER/asgs/output/cera_contour:/ddnA/work/$USER/asgs/output/cpra_postproc:/ddnA/work/$USER/asgs/output/cpra_postproc/MEX:/ddnA/work/$USER/asgs/output/cpra_postproc/old/tools:/ddnA/work/$USER/asgs/output/Cuba_post:/ddnA/work/$USER/asgs/output/FG49:/ddnA/work/$USER/asgs/output/NGOM_post:/ddnA/work/$USER/asgs/output/PartTrack:/ddnA/work/$USER/asgs/output/postProcessFlux:/ddnA/work/$USER/asgs/output/POSTPROC_KMZGIS:/ddnA/work/$USER/asgs/output/POSTPROC_KMZGIS/FigGen:/ddnA/work/$USER/asgs/output/POSTPROC_KMZGIS/RenciGETools-10/src:/ddnA/work/$USER/asgs/output/test:/ddnA/work/$USER/asgs/output/TRACKING_FILES:/ddnA/work/$USER/asgs/output/validation:/ddnA/work/$USER/asgs/tides:/ddnA/work/$USER/asgs/util:/ddnA/work/$USER/asgs/util/admin:/ddnA/work/$USER/asgs/util/input:/ddnA/work/$USER/asgs/util/input/mesh:/ddnA/work/$USER/asgs/util/input/nodalattr:/ddnA/work/$USER/asgs/util/output:/ddnA/work/$USER/asgs/util/troubleshooting:/ddnA/work/$USER/local-asgs-assets/bin:/usr/lib64/qt-3.3/bin:/usr/local/packages/mvapich2/2.3.3/axaxsfxv/bin:/usr/local/packages/Modules/4.2.5/bin:/usr/local/bin:/usr/bin:/usr/local/sbin:/usr/sbin:/usr/local/bin:/usr/local/compilers/Intel/parallel_studio_xe_2019.5/compilers_and_libraries/linux/bin/intel64/:/opt/ibutils/bin:/opt/dell/srvadmin/bin:/home/$USER/.local/bin:/home/$USER/bin",
    "env.adcirc.build.LD_LIBRARY_PATH"    : "/ddnA/work/$USER/asgs/opt/lib:/usr/local/packages/mvapich2/2.3.3/axaxsfxv/lib:/usr/local/compilers/Intel/parallel_studio_xe_2019.5/compilers_and_libraries/linux/lib/intel64:/usr/local/compilers/Intel/parallel_studio_xe_2019.5/mkl/lib/intel64",
    "env.adcirc.build.LD_INCLUDE_PATH"    : "/ddnA/work/$USER/asgs/opt/include:/usr/local/compilers/Intel/parallel_studio_xe_2019.5/mkl/include",
    "adcirc.build.fortran.mpif90"         : "/usr/local/packages/mvapich2/2.3.3/axaxsfxv/bin/mpif90",
    "adcirc.build.fortran.compiler"       : "/usr/local/compilers/Intel/parallel_studio_xe_2019.5/compilers_and_libraries/linux/bin/intel64/ifort",
    "adcirc.build.c.compiler"             : "/usr/local/compilers/Intel/parallel_studio_xe_2019.5/compilers_and_libraries/linux/bin/intel64/icc",
    "adcirc.build.c.compiler.version"     : "icc (ICC) 19.0.5.281 20190815",
    "adcirc.build.modules.loaded"         : " 1) intel/19.0.5   2) mvapich2/2.3.3/intel-19.0.5  "
    "adcirc.build.fortran.compiler.version" : "ifort (IFORT) 19.0.5.281 20190815",
  }
```

**Comment by wwlwpd** [comment]
@jasonfleming didn't you determine the `fort.`15 is not comforming to what this version of ADCIRC is expecting?

**Comment by jasonfleming** [comment]
My hunch at the time was that this is an issue with what this version of ADCIRC was expecting, but need to reproduce this to confirm and will record the result here (likely an update of the issue title as you suggest as well).

**Comment by jasonfleming** [comment]
This was resolved by #1055
